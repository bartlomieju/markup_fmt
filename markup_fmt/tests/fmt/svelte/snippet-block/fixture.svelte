{#snippet  product(p: Product)}
  <div >{ p.name}
  </div>
{/snippet}
