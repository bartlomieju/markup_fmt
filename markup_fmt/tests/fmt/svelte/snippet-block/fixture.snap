---
source: markup_fmt/tests/fmt.rs
---
{#snippet product(p: Product)}
  <div>
    {p.name}
  </div>
{/snippet}